### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by rical730**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,8 @@ Several-points to get you started:
 - [Advanced Tutorial](https://parl.readthedocs.io/en/latest/implementations/new_alg.html) : Create customized algorithms.
 - [API documentation](https://parl.readthedocs.io/en/latest/apis/model.html)
 
+We also provide an introductory course on reinforcement learning (RL) for zero-based developers : ( [Course](https://aistudio.baidu.com/aistudio/course/introduce/1335) | [Video](https://www.bilibili.com/video/BV1yv411i7xd) | [Code](examples/tutorials/) )
```
Comment (TomorrowIsAnOtherDay) [inline README.md]: zero-based developers -> absolute beginners.

Comment (TomorrowIsAnOtherDay) [reply README.md]: please remove the link towards the aistudio.

File: README.cn.md [hunk 2]
```diff
@@ -90,6 +90,8 @@ pip install parl
 - [进阶教程](https://parl.readthedocs.io/zh_CN/latest/implementations/new_alg.html) : 自定义新算法。
 - [API 文档](https://parl.readthedocs.io/zh_CN/latest/apis/model.html)
 
+同时，我们还为零基础开发者提供强化学习入门课程 : ( [课程](https://aistudio.baidu.com/aistudio/course/introduce/1335) | [视频](https://www.bilibili.com/video/BV1yv411i7xd) | [代码](examples/tutorials/) )
```
Comment (TomorrowIsAnOtherDay) [inline README.cn.md]: please remove the link towards the AIStudio.

File: examples/DQN_variant/readme.md [hunk 3]
```diff
@@ -54,4 +54,4 @@ python train.py --rom ./rom_files/pong.bin
 python train.py --help
 ```
 
-> To train more games, you can install more rom files from [here](https://github.com/openai/atari-py/tree/master/atari_py/atari_roms).
+> To train more games, you can install more rom files from [here](https://github.com/openai/atari-py/tree/s3/atari_py/atari_roms).
```
Comment (TomorrowIsAnOtherDay) [inline examples/DQN_variant/readme.md:57]: Can you update the link for torch-based implementation ?

Comment (rical730) [reply examples/DQN_variant/readme.md:57]: will do after torch_dqn updated

File: README.md (hunk 1 above)
Comment (rical730) [reply README.md]: done

File: README.cn.md (hunk 2 above)
Comment (rical730) [reply README.cn.md]: done

**Review by TomorrowIsAnOtherDay — DISMISSED** [review]

**Review by TomorrowIsAnOtherDay — APPROVED** [review]

**Pull request merged**